internal/controller/stas/scan_job_controller.go

```diff
@@ -169,6 +171,8 @@ func (r *ScanJobReconciler) reconcile() reconcile.Func {
 func (r *ScanJobReconciler) reconcileJob(ctx context.Context, job *batchv1.Job) error {
 	cisList := &stasv1alpha1.ContainerImageScanList{}
 
+	logf.FromContext(ctx).V(3).Info("Reconciling job(%s) with status %+v", job.Name, job.Status)
```

Comment:
```suggestion
	logf.FromContext(ctx).V(3).Info("Reconciling job %q with status %+v", job.Name, job.Status)
```

And what does the `+` in `%+v` do?

Reply:
when printing structs, the plus flag adds field names